src/components/Blankslate/Blankslate.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import { Icon } from '../Icon/Icon';
+
+import classNames from 'classnames';
+
+export interface BlankslateProps {
+	title: string;
+	body: string;
+	spacious?: boolean;
```

Comment:
default: false
required: false
icon is also required: false

![image](https://user-images.githubusercontent.com/1710840/58540188-28088e80-81f9-11e9-8dab-fb15a8481bf4.png)

Reply:
Da's gewoon de manier waarop react-docgen werkt, als het required is staat er `true`, anders staat er `-` (I know, `false` was logischer geweest). Hetzelfde met `default`, geen default value = `-`.